src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/DigitalObjects/DigitalObjectRepository.cs

```diff
@@ -578,37 +589,77 @@ private List<Image> GetOrphans(Dictionary<string, Image> imagesAlreadyOnDlcs, IE
+        private Image? GetIngestDiffImage(Image newDlcsImage, Image existingDlcsImage, out List<string> reasons)
         {
+            reasons = new List<string>();
             Image? reingestImage = null;
-            const string reingestMessageFormat =
+            const string reingestMessageStructuredLoggingFormat =
                 "Reingest required for {identifier}. Mismatch for {field} - new: {newValue}, existing: {existingValue}";
+            const string reingestMessageUIFormat =
+                "REINGEST because mismatch for {0} - new: '{1}', existing: '{2}'";
             if (existingDlcsImage.Origin != newDlcsImage.Origin)
             {
-                logger.LogDebug(reingestMessageFormat, newDlcsImage.StorageIdentifier, "origin", newDlcsImage.Origin,
-                    existingDlcsImage.Origin);
+                logger.LogDebug(reingestMessageStructuredLoggingFormat, newDlcsImage.StorageIdentifier, 
+                    "origin", newDlcsImage.Origin, existingDlcsImage.Origin);
+                reasons.Add(string.Format(reingestMessageUIFormat,
+                    "origin", newDlcsImage.Origin, existingDlcsImage.Origin));
                 reingestImage ??= newDlcsImage;
             }
 
             // This one CURRENTLY requires reingest because it's engine that modifies the S3 location
             // But later it could go into regular batch-patch if protagonist no longer rejects it.
             if (existingDlcsImage.MaxUnauthorised != newDlcsImage.MaxUnauthorised)
             {
-                logger.LogDebug(reingestMessageFormat, newDlcsImage.StorageIdentifier, "maxUnauthorised",
-                    newDlcsImage.MaxUnauthorised, existingDlcsImage.MaxUnauthorised);
+                logger.LogDebug(reingestMessageStructuredLoggingFormat, newDlcsImage.StorageIdentifier, 
+                    "maxUnauthorised", newDlcsImage.MaxUnauthorised, existingDlcsImage.MaxUnauthorised);
+                reasons.Add(string.Format(reingestMessageUIFormat,
+                    "maxUnauthorised", newDlcsImage.MaxUnauthorised, existingDlcsImage.MaxUnauthorised));
                 reingestImage ??= newDlcsImage;
             }
 
             // In DDS we never change these but they would need to be considered here
             // We also need to consider what's significant when either new or existing are null
-            //if (existingDlcsImage.ImageOptimisationPolicy != newDlcsImage.ImageOptimisationPolicy)
-            //    return newDlcsImage;
-            //if (existingDlcsImage.ThumbnailPolicy != newDlcsImage.ThumbnailPolicy)
-            //    return newDlcsImage;
-            //if (existingDlcsImage.DeliveryChannel != newDlcsImage.DeliveryChannel)
-            //    return newDlcsImage;
+            if (dlcs.SupportsDeliveryChannels)
+            {
+                // While we now set IOP to `use-original` for JP2s, we're relying on DLCS defaults for everything else still.
```

Comment:
Are there tickets to implement the commented out iop + thumbs stuff?

Reply:
I think this is captured here - https://github.com/dlcs/protagonist/issues/489